src/main/resources/assets/carpet/scripts/distance.sc

```diff
@@ -0,0 +1,55 @@
+import('math','_euclidean','_manhattan','_round');
+
+__config()->{
+    'commands'->{
+        'from <start_pos>'->'set_start',
+        'from <start_pos> to <end_pos>'->'_calculate',
+        'to <end_pos>'->_(end_pos)->(_calculate(null,end_pos))
+    }
+};
+
+global_positions={};
+
+set_start(pos)->(
+    global_positions:player() = pos;
+    print(player(),format('gi Initial point set to: ', 'g '+pos,'?/tp '+(str(pos)-','-'['-']')))
+);
+
+_calculate(start_pos, end_pos)->(
+    if(start_pos!=null,
+        set_start(start_pos);
+        calculate(end_pos),
+        if(has(global_positions,player()),
+            calculate(end_pos),
+            print(player(),format('r There was no initial point for '+player()));
+            set_start(end_pos)
+        )
+    )
+);
+
+__on_player_places_block(player, item_tuple, hand, block) ->(
+    if(system_info('world_carpet_rules'):'carpets'&&block=='brown_carpet',
```

Comment:
`world_carpet_rules` always returns strings, so you need a string equals `'true'` check instead.

Reply:
ah

Reply:
Ill test.

Reply:
but tomorrow